api/services.py

```diff
@@ -0,0 +1,35 @@
+from fastapi import Depends
+from sqlalchemy import text
+from sqlalchemy.orm import Session
+from apscheduler.schedulers.background import BackgroundScheduler
+from api.routes import get_price_levels, get_db
+
+def when_to_notify(zone: str, db: Session = Depends(get_db)):
+    # this gets the high price of the last month
+    price_levels = get_price_levels(zone=zone, db=db)
+    high_price_last_month = price_levels["priceLevels"]["high"]
+
+    # this gets the current price
+    query_current_price = text("""
+            SELECT price_sek FROM price_data 
+            WHERE zone = :zone 
+            ORDER BY time_start DESC 
```

Comment:
Our aggregator gets data for each day, 0-24 hours all at once and day-ahead, so in the morning the prices for the whole day is known. I think this causes this query to always only look at the price at the end of the day. The simplest solution to this would be to order somehow by how close the time_start is to the current time.